mlflow/models/evaluation/base.py

```diff
@@ -1278,7 +1291,13 @@ def evaluate(
           specified, unless the ``evaluator_config`` option **log_model_explainability** is
           explicitly set to ``True``.
 
-    :param model: A pyfunc model instance, or a URI referring to such a model.
+    :param model: One of the following:
+
+                  - A pyfunc model instance
+
+                  - A URI referring to a pyfunc model
+
+                  - A function
```

Comment:
Can we indicate which output types from the function are supported / point to a reference?

Reply:
Use the same wording as here: https://mlflow.org/docs/latest/python_api/mlflow.pyfunc.html#inference-api